src/Model/AuditReader.php

```diff
@@ -47,7 +47,7 @@ public function find(string $className, $id, $revisionId): ?object
         }
     }
 
-    public function findRevisionHistory(string $className, ?int $limit = 20, ?int $offset = 0): array
+    public function findRevisionHistory(string $className, int $limit = 20, int $offset = 0): array
```

Comment:
@VincentLanglet the interface does not have the "?" on both parameters. Is it a BC break to remove them here?

Reply:
To me it's a BC break indeed.
```
AuditReader::findRevisionHistory(Foo::class, null, null)
```
is working and won't after your PR.

Reply:
We will need to find out a way to fix phpstan then, because it is failing because it thinks the method can accept null.

Reply:
Take a look at https://github.com/sonata-project/SonataAdminBundle/pull/7959